src/PrestaShopBundle/Form/Admin/Improve/International/Localization/ImportLocalizationPackType.php

```diff
@@ -72,6 +72,10 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ),
                 'choices' => $this->localizationPackChoices,
                 'choice_translation_domain' => false,
+                'attr' => [
+                    'data-minimumResultsForSearch' => '7',
```

Comment:
Why 7 ? and not a constant ?

Reply:
This is the same thing used inside the whole page... I just copied the rest of the page (and verified that it's the right value)